linux/keyman-system-service/src/OrderOutputDevice.cpp

```diff
@@ -0,0 +1,74 @@
+#include <dirent.h>
+#include <fcntl.h>
+#include <iostream>
+#include <string.h>
+#include <string>
+#include <syslog.h>
+#include <unistd.h>
+#include "OrderOutputDevice.h"
+
+using namespace std;
+
+#define KEYMAN_F24_KEYCODE_OUTPUT_SENTINEL 194  // 0xC2
```

Comment:
Ideally we should be sharing value with ibus-keyman rather than defining in two places